scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -349,6 +351,19 @@ public boolean kill() {
 
         listener.schedulerStateUpdated(SchedulerEvent.KILLED);
 
+        // To properly exit the java scheduling process
+        new Thread(new Runnable() {
```

Comment:
You could use lambda. Idea can help you to change it)

Reply:
Ok thx!